tree/ntuple/src/RFieldUtils.cxx

```diff
@@ -531,6 +532,39 @@ std::string ROOT::Internal::GetRenormalizedTypeName(const std::string &metaNorma
    return GetRenormalizedMetaTypeName(metaNormalizedName);
 }
 
+bool ROOT::Internal::NeedsMetaNameAsAlias(const std::string &metaNormalizedName, std::string &renormalizedAlias)
```

Comment:
This seems similar to (but better than): `TClassEdit::GetLong64_Name`, isn't it?

Reply:
Perhaps. This routine goes through the same machinery than the "normal" RNTuple type normalization, just that it fixes up instances of `[U]Long64_t` in template parameters where necessary.